cooker/lib/pages.js

```diff
@@ -244,6 +244,9 @@ async function pages(file, localeFile) {
     // The data file for MD. This just includes any object data within an MD file, if that's what we're compiling
     const mdData = {};
 
+    // getting the pathname for globals
+    let pathname = `/${path.relative(`${config.CWD}/pages`, templateInfo.dir)}`;
```

Comment:
Take a look at [path.resolve](https://nodejs.org/api/path.html#path_path_resolve_paths) it might be better suited for this. Also, please use the the [config.PAGES_DIRECTORY](https://github.com/wethegit/sweet-potato/blob/main/utensils/lib/config.js#L59) variable so we make sure that we are always referencing the correct folder in case it changes in the future or on a user configuration